roles/sap_hypervisor_node_preconfigure/tasks/platform/redhat_ocp_virt/install-sriov-operator.yml

```diff
@@ -0,0 +1,55 @@
+---
+- name: Create the CNV Operator namespace
+  kubernetes.core.k8s:
+    state: present
+    definition:
+      apiVersion: v1
+      kind: Namespace
+      metadata:
+        name: openshift-sriov-network-operator
+
+- name: Create the CNV Operator namespace
+  kubernetes.core.k8s:
+    state: present
+    definition:
+      apiVersion: operators.coreos.com/v1
+      kind: OperatorGroup
+      metadata:
+        name: sriov-network-operators
+        namespace: openshift-sriov-network-operator
+      spec:
+        targetNamespaces:
+          - openshift-sriov-network-operator
+- name: Create the CNV Operator namespace
+  kubernetes.core.k8s:
+    state: present
+    definition:
+      apiVersion: operators.coreos.com/v1alpha1
+      kind: Subscription
+      metadata:
+        name: sriov-network-operator-subscription
+        namespace: openshift-sriov-network-operator
+      spec:
+        source: redhat-operators
+        sourceNamespace: openshift-marketplace
+        name: sriov-network-operator
+      #  startingCSV: sriov-network-operator
+        channel: "stable"
+
+
+- name: Pause to give operator a chance to install
+  ansible.builtin.pause:
+    minutes: 3
+
+- name: Copy patch to enable unsupported NICs
```

Comment:
As mentioned above, use kubernetes.core.k8s with merge_type: merge or kubernetes.core.k8s_json_patch instead of oc? Why are a separate script, copy and command modules required?